### PR TITLE
Allow to refine the request on EventBrite imports

### PR DESCRIPTION
**Pull request by mitogh**
Ticket: https://central.tri.be/issues/120405
Ticket: https://central.tri.be/issues/137821

Allow the user the option to refine the import when importing from "your account".

:video_camera: http://p.tri.be/zxWtOJ

### CODE REVIEW

File: src/admin-views/aggregator/origins/refine.php [hunk 1]
```diff
@@ -9,6 +9,12 @@
 $radius->placeholder     = sprintf( _x( 'Radius (%s)', 'Radius with abbreviation', 'the-events-calendar' ), Tribe__Events__Utils__Radius::get_abbreviation() );
 $depends_condition       = 'data-condition-not-empty';
 
+$location_depends = '#tribe-ea-field-origin';
+$keyword_depends = '#tribe-ea-field-origin';
+
+$keyword_exclusions = ['facebook'];
+$location_exclusions = ['url', 'facebook'];
```
Comment (bordoni) [inline src/admin-views/aggregator/origins/refine.php]: Spacing around `[]`
```suggestion
$location_exclusions = [ 'url', 'facebook' ];
```

File: src/admin-views/aggregator/origins/refine.php [hunk 2]
```diff
@@ -9,6 +9,12 @@
 $radius->placeholder     = sprintf( _x( 'Radius (%s)', 'Radius with abbreviation', 'the-events-calendar' ), Tribe__Events__Utils__Radius::get_abbreviation() );
 $depends_condition       = 'data-condition-not-empty';
 
+$location_depends = '#tribe-ea-field-origin';
+$keyword_depends = '#tribe-ea-field-origin';
+
+$keyword_exclusions = ['facebook'];
```
Comment (bordoni) [inline src/admin-views/aggregator/origins/refine.php]:
```suggestion
$keyword_exclusions = [ 'facebook' ];
```

File: src/admin-views/aggregator/origins/refine.php [hunk 3]
```diff
@@ -39,32 +50,24 @@
 		break;
 }
 
-/**
- * Allow filtering of origins excluded from refining EA results by keyword.
- *
- * @since 4.6.24
- *
- * @param array $keyword_exclusions List of origins excluded.
- */
```
Comment (bordoni) [inline src/admin-views/aggregator/origins/refine.php]: Dont exclude the Docblock.

Comment (mitogh) [reply src/admin-views/aggregator/origins/refine.php]: Woops my bad.

**Pull request merged**